libs/imageio/src/ImageEncoder.cpp

```diff
@@ -249,6 +255,8 @@ std::string ImageEncoder::chooseExtension(ImageEncoder::Format format) {
         case Format::PNG:
         case Format::PNG_LINEAR:
             return ".png";
+        case Format::RGB_10_11_11_REV:
+            return ".rgb32f";
```

Comment:
The name is a bit misleading, it sounds like it would be a 32 bit float per channel.

Reply:
I'm open to better suggestions. I didn't like .rgb111110 :-)

Reply:
Maybe just `rgb11f`?